front-app/src/api/usersApi.tsx

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import { RegisterValidationInterface } from '../views/Register';
+import { LoginValidationInterface } from '../views/Login';
+
+const usersApi = axios.create({
+  baseURL: 'http://localhost:4000/users',
+});
+
+export const getUsers = async () => {
+  const response = await usersApi.get('/');
+  return response.data;
+};
```

Comment:
Remember we almost always prefer this sin tax in this kind of situations

```suggestion
export const getUsers = async () => {
  const { data } = await usersApi.get('/');
  return data;
};
```